cannon/mipsevm/exec/mips_syscalls.go

```diff
@@ -132,7 +133,18 @@ const (
 
 // Other constants
 const (
+	// SchedQuantum is the number of steps dedicated for a thread before it's preempted. Effectively used to emulate thread "time slices"
 	SchedQuantum = 100_000
+
+	// HZ is the assumed clock rate of an emulated MIPS32 CPU.
+	// The value of HZ is a rough estimate of the Cannon instruction count / second on a typical machine.
+	// HZ is used to emulate the clock_gettime syscall used by guest programs that have a Go runtime.
+	// The Go runtime consumes the system time to determine when to initiate gc assists and for goroutine scheduling.
+	// A HZ value that is too low (i.e. lower than the emulation speed) results in the main goroutine attempting to assist with GC more often.
+	// Adjust this value accordingly as the emulation speed changes. The HZ value should be within the same order of magnitude as the emulation speed.
```

Comment:
When the main goroutine assists in GC, it may not actually perform a GC. Or I should say that the main goroutine _attempts_ to assist. There are other factors including memory pressure, and last time since the last GC, that affect when a GC actually occurs. As such, we can't rely on `runtime` metrics like `NumGC` to figure out if this is occurring to frequently.

A GC assist attempt that doesn't actual run the gc isn't expensive (about 1-5% of an actual GC run from what I've seen). So I'm not too worried if the op-program attempts to assist more often that necessary. As long as it doesn't do so for a large fraction of memory allocations. And I assume it'll be pretty evident once we're seeing alot of assist attempts because the runtime would be noticeably slower.